editor/debugger/editor_profiler.cpp

```diff
@@ -751,15 +756,15 @@ EditorProfiler::EditorProfiler() {
 	variables->set_hide_root(true);
 	variables->set_columns(3);
 	variables->set_column_titles_visible(true);
-	variables->set_column_title(0, TTR("Name"));
+	variables->set_column_title(0, TTRC("Name"));
 	variables->set_column_expand(0, true);
 	variables->set_column_clip_content(0, true);
 	variables->set_column_custom_minimum_width(0, 60);
-	variables->set_column_title(1, TTR("Time"));
+	variables->set_column_title(1, TTRC("Time"));
 	variables->set_column_expand(1, false);
 	variables->set_column_clip_content(1, true);
 	variables->set_column_custom_minimum_width(1, 75 * EDSCALE);
-	variables->set_column_title(2, TTR("Calls"));
+	variables->set_column_title(2, TTRC("Calls"));
```

Comment:
`variables` is in the same situation as `monitor_tree` in the performance profiler.

We may want to enable auto translation for the `Tree` and disable for each `TreeItem`.